.github/workflows/go.yml

```
@@ -16,17 +16,46 @@ jobs:
+        
+        # Rename extracted folder to match expected path
+        move C:\oracle_inst\instantclient_23_7 C:\oracle_inst\instantclient_23_7_temp || true
+        move C:\oracle_inst\instantclient_* C:\oracle_inst\instantclient_23_7 || true
+      shell: pwsh
+
+    - name: Add Oracle Instant Client to PATH
+      run: echo "C:\oracle_inst\instantclient_23_7" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+      shell: pwsh
+
     - name: Set up Go
       uses: actions/setup-go@v4
```

actionlint (1.7.9)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)